src/tileset.rs

```diff
@@ -37,10 +37,6 @@ pub struct Tileset {
 
     /// The custom properties of the tileset.
     pub properties: Properties,
-
-    /// Where this tileset was loaded from.
-    /// If fully embedded, this will return `None`.
-    pub source: Option<PathBuf>,
```

Comment:
I'd really like to get rid of it for a few reasons, but you're right, until we get embedded fully figured out it doesn't make much sense to delete it.

My reasons are:
- The `Some` variant is duplicate data already found in the respective cache or provided by the user via `new_external`.
- IMO, The tileset source should be irrelevant to its data. It makes no difference externally, and the only reason it *was* required was to associate tilesets with paths to be cached in a hashmap. This is no longer required now since we already provide our own resource cache.
- Tests for tileset equality between embedded/external tilesets can't just check for equality in the entire struct, and have to test for each member.

Reply:
Alright, let's just yank it, since actually the use-case of differentiating between external and embedded tilesets doesn't currently exist. It would mostly only be relevant when we implement modifications.

Reply:
Only now have I seen this. :p